test/e2e/telemeter_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2019 The Cluster Monitoring Operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/openshift/cluster-monitoring-operator/pkg/manifests"
+	"github.com/pkg/errors"
+)
+
+func TestTelemeterTrustedCA(t *testing.T) {
+	var (
+		factory = manifests.NewFactory("openshift-monitoring", nil)
+		newCM   *v1.ConfigMap
+		lastErr error
+	)
+
+	// Wait for Telemeter
+	err := wait.Poll(time.Second, 5*time.Minute, func() (bool, error) {
```

Comment:
hmm not a big deal but just realized that we are waiting for the telemeter-client so we will only catch this on the second run, as we create the Trusted CA bundle ConfigMap before we create the deployment. I think we should be waiting for the `telemeter-trusted-ca-bundle` first.